transport-native-epoll/src/main/c/netty_unix_socket.c

```diff
@@ -626,76 +629,94 @@ static jint netty_unix_socket_getSoLinger(JNIEnv* env, jclass clazz, jint fd) {
-  { "getSoError", "(I)I", (void *) netty_unix_socket_getSoError },
-  { "getTcpDeferAccept", "(I)I", (void *) netty_unix_socket_getTcpDeferAccept },
-  { "isTcpQuickAck", "(I)I", (void *) netty_unix_socket_isTcpQuickAck }
+    { "shutdown", "(IZZ)I", (void *) netty_unix_socket_shutdown},
+    { "bind", "(I[BII)I", (void *) netty_unix_socket_bind},
+    { "listen", "(II)I", (void *) netty_unix_socket_listen},
+    { "connect", "(I[BII)I", (void *) netty_unix_socket_connect},
+    { "finishConnect", "(I)I", (void *) netty_unix_socket_finishConnect},
+    { "accept", "(I[B)I", (void *) netty_unix_socket_accept},
+    { "remoteAddress", "(I)[B", (void *) netty_unix_socket_remoteAddress},
+    { "localAddress", "(I)[B", (void *) netty_unix_socket_localAddress},
+    { "newSocketDgramFd", "()I", (void *) netty_unix_socket_newSocketDgramFd},
+    { "newSocketStreamFd", "()I", (void *) netty_unix_socket_newSocketStreamFd},
+    { "newSocketDomainFd", "()I", (void *) netty_unix_socket_newSocketDomainFd},
+    { "sendTo", "(ILjava/nio/ByteBuffer;II[BII)I", (void *) netty_unix_socket_sendTo},
+    { "sendToAddress", "(IJII[BII)I", (void *) netty_unix_socket_sendToAddress},
+    { "sendToAddresses", "(IJI[BII)I", (void *) netty_unix_socket_sendToAddresses},
+    // "recvFrom" has a dynamic signature
+    // "recvFromAddress" has a dynamic signature
+    { "bindDomainSocket", "(I[B)I", (void *) netty_unix_socket_bindDomainSocket},
+    { "connectDomainSocket", "(I[B)I", (void *) netty_unix_socket_connectDomainSocket},
+    { "setTcpNoDelay", "(II)V", (void *) netty_unix_socket_setTcpNoDelay},
+    { "setReceiveBufferSize", "(II)V", (void *) netty_unix_socket_setReceiveBufferSize},
+    { "setSendBufferSize", "(II)V", (void *) netty_unix_socket_setSendBufferSize},
+    { "setKeepAlive", "(II)V", (void *) netty_unix_socket_setKeepAlive},
+    { "setTcpCork", "(II)V", (void *) netty_unix_socket_setTcpCork},
+    { "setSoLinger", "(II)V", (void *) netty_unix_socket_setSoLinger},
+    { "setTcpDeferAccept", "(II)V", (void *) netty_unix_socket_setTcpDeferAccept},
+    { "setTcpQuickAck", "(II)V", (void *) netty_unix_socket_setTcpQuickAck},
+    { "isKeepAlive", "(I)I", (void *) netty_unix_socket_isKeepAlive},
+    { "isTcpNoDelay", "(I)I", (void *) netty_unix_socket_isTcpNoDelay},
+    { "getReceiveBufferSize", "(I)I", (void *) netty_unix_socket_getReceiveBufferSize},
+    { "getSendBufferSize", "(I)I", (void *) netty_unix_socket_getSendBufferSize},
+    { "isTcpCork", "(I)I", (void *) netty_unix_socket_isTcpCork},
+    { "getSoLinger", "(I)I", (void *) netty_unix_socket_getSoLinger},
+    { "getSoError", "(I)I", (void *) netty_unix_socket_getSoError},
+    { "getTcpDeferAccept", "(I)I", (void *) netty_unix_socket_getTcpDeferAccept},
+    { "isTcpQuickAck", "(I)I", (void *) netty_unix_socket_isTcpQuickAck},
+    { "peerCredentials", "(ILio/netty/channel/unix/PeerCredentials;)I", (void *) netty_channel_unix_socket_peerCredentials}
```

Comment:
this signature is "dynamic" in that is include a Netty type in the signature. Because we support shading the type signature is not fixed and should be built in the `createDynamicMethodsTable` method. Also update the size in `dynamicMethodsTableSize`.